tmpfiles.d/systemd-remote.conf

```diff
@@ -7,5 +7,8 @@
 
 # See tmpfiles.d(5) for details
 
-z /var/log/journal/remote 2755 root systemd-journal-remote - -
-z /run/log/journal/remote 2755 root systemd-journal-remote - -
+z /var/log/journal/remote 2755 systemd-journal-remote systemd-journal-remote - -
+z /run/log/journal/remote 2755 systemd-journal-remote systemd-journal-remote - -
+
+a+ /var/log/journal/remote - - - - d:group:adm:r-x,d:group:wheel:r-x
+a+ /var/log/journal/remote - - - - group:adm:r-x,group:wheel:r-x
```

Comment:
Never mind, we set recursive acls on /var/log/journal. So we don't need to duplicate the acl setting on /var/log/journal/remote. This dir would be created after acls were already applied on /var/log/journal, so it should get the right acls. But the ownership should be fixed indeed.

@